<html>
<head>
  <title>ggd-genomics</title>
  <style type="text/css">
    a, a:active {
      text-decoration: none; color: blue;
    }
    a:visited {
      color: #48468F;
    }
    a:hover, a:focus {
      text-decoration: underline; color: red;
    }
    body {
      background-color: #F5F5F5;
    }
    h2 {
      margin-bottom: 12px;
    }
    th, td {
      font: 100% monospace; text-align: left;
    }
    th {
      font-weight: bold; padding-right: 14px; padding-bottom: 3px;
    }
    th.tight {
        padding-right: 6px;
    }
    td {
      padding-right: 14px;
    }
    td.tight {
        padding-right: 8px;
    }
    td.s, th.s {
      text-align: right;
    }
    td.summary {
      white-space: nowrap;
      overflow: hidden;
    }
    td.packagename {
      white-space: nowrap;
      text-overflow: ellipsis;
      overflow: hidden;
      max-width: 180px;
      padding-right: 8px;
    }
    td.version {
      //white-space: nowrap;
      overflow: hidden;
      max-width: 90px;
      padding-right: 8px;
    }
    table {
      background-color: white;
      border-top: 1px solid #646464;
      border-bottom: 1px solid #646464;
      padding-top: 10px;
      padding-bottom: 14px;
    }
    address {
      color: #787878;
      padding-top: 10px;
    }
  </style>
</head>
<body>
  <h2>ggd-genomics</h2>
  <h3><a href="rss.xml">RSS Feed</a>&nbsp;&nbsp;&nbsp;<a href="channeldata.json">channeldata.json</a></h3>
<a href="noarch">noarch</a>&nbsp;&nbsp;&nbsp;  <table>
    <tr>
      <th style="padding-right:18px;">Package</th>
      <th>Latest Version</th>
      <th>Doc</th>
      <th>Dev</th>
      <th>License</th>
<th class="tight">noarch</th>      <th>Summary</th>
    </tr>
    <tr>
      <td class="packagename">danrer10-gtf-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCz10 gtf file from ensembl remapped to danRer10 (Any unmapped...</td>
    </tr>    <tr>
      <td class="packagename">danrer10-gtf-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">danrer10-reference-genome-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The danRer10 reference genome from UCSC</td>
    </tr>    <tr>
      <td class="packagename">danrer11-reference-genome-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The danRer11 reference genome from UCSC</td>
    </tr>    <tr>
      <td class="packagename">grch37-1000g-omni-snps-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The 1000G omni snps vcf file</td>
    </tr>    <tr>
      <td class="packagename">grch37-1000g-snps-phase1-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The 1000G high confidence snps from phase 1 in vcf format</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-dominant-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genomic coordinates, along with the compliment coordinance, for...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-dominant-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genomic coordinates, along with the compliment coordinance, from OMIM...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-dominant-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genomic coordinates, along with the compliment coordinance, OMIM...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-recessive-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genomic coordinates, along with the compliment coordinance, for...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-recessive-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genomic coordinates, along with the compliment coordinance, from OMIM...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-recessive-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genomic coordinates, along with the compliment coordinance, OMIM...</td>
    </tr>    <tr>
      <td class="packagename">grch37-chrom-mapping-ucsc2ensembl-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps hg19 UCSC...</td>
    </tr>    <tr>
      <td class="packagename">grch37-chromsizes-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for GRCh37</td>
    </tr>    <tr>
      <td class="packagename">grch37-chromsizes-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for the GRCh37 genome build from NCBI. (Used to...</td>
    </tr>    <tr>
      <td class="packagename">grch37-cpg-islands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">cpg islands from UCSC in bed format. Remapped from UCSC hg19 to Ensembl...</td>
    </tr>    <tr>
      <td class="packagename">grch37-cytobands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Approximate locations of Chromosome Bands (cytoBands) located with...</td>
    </tr>    <tr>
      <td class="packagename">grch37-dbsnp-138-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The dbsnp version 138 vcf file</td>
    </tr>    <tr>
      <td class="packagename">grch37-eiee-genes-ostrander-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">This is a set of genes from a recent paper on EIEE (Early Infantile...</td>
    </tr>    <tr>
      <td class="packagename">grch37-esp-variants-uw-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The NHLBI Exome Sequencing Project (ESP) variants from the University...</td>
    </tr>    <tr>
      <td class="packagename">grch37-evofold-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The evolutionary conserved mRNA secondary structure predictions hosted...</td>
    </tr>    <tr>
      <td class="packagename">grch37-gaps-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Assembly gaps from UCSC in bed fromat. Remapped from UCSC hg19 to...</td>
    </tr>    <tr>
      <td class="packagename">grch37-gtf-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh37 gtf file from ensembl</td>
    </tr>    <tr>
      <td class="packagename">grch37-haploinsufficient-genes-clingen-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The &#39;clingen_hi_gene&#39; haploinsufficient gene set comes from...</td>
    </tr>    <tr>
      <td class="packagename">grch37-haploinsufficient-genes-dang-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The haploinsufficient gene set comes from Dang et al. (2008):...</td>
    </tr>    <tr>
      <td class="packagename">grch37-hapmap-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hapmap vcf versoin 3.3 from the broad institute</td>
    </tr>    <tr>
      <td class="packagename">grch37-heart-genes-sifrim-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Manually curated list of congenital heart disease (CHD) genes from...</td>
    </tr>    <tr>
      <td class="packagename">grch37-hg002-pb-chr22-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">GiaB PacBio 70x sequencing for chromosome 22 in ashkenazi sample HG002</td>
    </tr>    <tr>
      <td class="packagename">grch37-hg003-pb-chr22-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">GiaB PacBio 30x sequencing for chromosome 22 in ashkenazi sample HG003</td>
    </tr>    <tr>
      <td class="packagename">grch37-hg004-pb-chr22-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">GiaB PacBio 30x sequencing for chromosome 22 in ashkenazi sample HG004</td>
    </tr>    <tr>
      <td class="packagename">grch37-microsatellites-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Microsatellites from UCSC. Region which tend to be highly polymorphic...</td>
    </tr>    <tr>
      <td class="packagename">grch37-pfam-domains-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">High quality, manually curated Pfam domain annotation in bed12 format...</td>
    </tr>    <tr>
      <td class="packagename">grch37-phase3-vcf-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The phase3 1000g vcf file distrubted on 05/02/2013 in vcf format...</td>
    </tr>    <tr>
      <td class="packagename">grch37-reference-genome-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">GRCh37 reference genome from 1000 genomes</td>
    </tr>    <tr>
      <td class="packagename">grch37-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh37 reference genome from Ensembl. Release 75. Primary Assembly file</td>
    </tr>    <tr>
      <td class="packagename">grch37-repeatmasker-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">RepeatMasker track from UCSC in bed format. Interspersed repeats and...</td>
    </tr>    <tr>
      <td class="packagename">grch37-scores-by-amino-acid-change-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">missense badnass, blosum, grantham scores foreach amino acid change</td>
    </tr>    <tr>
      <td class="packagename">grch37-segmental-dups-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Segmental duplications (SuperDups) File from UCSC. That is,...</td>
    </tr>    <tr>
      <td class="packagename">grch37-sequence-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The fasta sequence from 1000g</td>
    </tr>    <tr>
      <td class="packagename">grch37-simple-repeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC. Simple tandem repeats and imperfect...</td>
    </tr>    <tr>
      <td class="packagename">grch37-sv-gnomad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset from gnomAD</td>
    </tr>    <tr>
      <td class="packagename">grch37-vcf-2.1-exome-gnomad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The gnomAD whole exome vcf file. Version 2.1.1 released on March 6,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-chrom-mapping-ucsc2ensembl-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that map hg38 UCSC...</td>
    </tr>    <tr>
      <td class="packagename">grch38-chromsizes-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for GRCh38</td>
    </tr>    <tr>
      <td class="packagename">grch38-chromsizes-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for the GRCh38 genome build from NCBI. (Used to...</td>
    </tr>    <tr>
      <td class="packagename">grch38-cpg-islands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">cpg islands from UCSC in bed format. Remapped from UCSC hg38 to Ensembl...</td>
    </tr>    <tr>
      <td class="packagename">grch38-cytobands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Approximate locations of Chromosome Bands (cytoBands) located with...</td>
    </tr>    <tr>
      <td class="packagename">grch38-gaps-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Assembly gaps from UCSC in bed fromat Remapped from UCSC hg38 to...</td>
    </tr>    <tr>
      <td class="packagename">grch38-gtf-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh38 gtf file from ensembl</td>
    </tr>    <tr>
      <td class="packagename">grch38-hg002-chr22-illumina-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Illumina paired-end sequencing alignments from HG002, downloaded from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-hg003-chr22-illumina-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Illumina paired-end sequencing alignments from HG003, downloaded from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-hg004-chr22-illumina-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Illumina paired-end sequencing alignments from HG004, downloaded from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-microsatellites-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Microsatellites from UCSC. Region which tend to be highly polymorphic...</td>
    </tr>    <tr>
      <td class="packagename">grch38-pfam-domains-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">High quality, manually curated Pfam domain annotation in bed12 format...</td>
    </tr>    <tr>
      <td class="packagename">grch38-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh38 reference genome from Ensembl. Release 95. Primary Assembly file</td>
    </tr>    <tr>
      <td class="packagename">grch38-repeatmasker-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">RepeatMasker track from UCSC in bed format. Interspersed repeats and...</td>
    </tr>    <tr>
      <td class="packagename">grch38-segmental-dups-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Segmental duplications (SuperDups) File from UCSC. That is,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-simple-repeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC. Simple tandem repeats and imperfect...</td>
    </tr>    <tr>
      <td class="packagename">grcz10-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCz10 release 91 reference genome from Ensembl</td>
    </tr>    <tr>
      <td class="packagename">grcz11-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCz11 release 96 reference genome from Ensembl</td>
    </tr>    <tr>
      <td class="packagename">hg19-1000g-omni-snps-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The 1000G omni snps vcf file hosted on the broad&#39;s ftp site</td>
    </tr>    <tr>
      <td class="packagename">hg19-1000g-snps-phase1-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The 1000G high confidence snps from phase 1 in vcf format. (Hosted on...</td>
    </tr>    <tr>
      <td class="packagename">hg19-chrom-mapping-ensembl2ucsc-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps GRCh37...</td>
    </tr>    <tr>
      <td class="packagename">hg19-chromsizes-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for hg19</td>
    </tr>    <tr>
      <td class="packagename">hg19-chromsizes-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for the hg19 genome build from NCBI. (Used to create...</td>
    </tr>    <tr>
      <td class="packagename">hg19-cpg-islands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">cpg islands from UCSC in bed format. Scaffoldings that are not...</td>
    </tr>    <tr>
      <td class="packagename">hg19-cytobands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Approximate locations of Chromosome Bands (cytoBands) located with...</td>
    </tr>    <tr>
      <td class="packagename">hg19-dbsnp-138-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The dbsnp version 138 vcf file (hosted on the broad&#39;s ftp site)</td>
    </tr>    <tr>
      <td class="packagename">hg19-evofold-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The evolutionary conserved mRNA secondary structure predictions hosted...</td>
    </tr>    <tr>
      <td class="packagename">hg19-gaps-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Assembly gaps from UCSC in bed fromat. Scaffoldings that are not...</td>
    </tr>    <tr>
      <td class="packagename">hg19-gtf-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">hg19-hapmap-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hapmap vcf versoin 3.3 (hosted on the  broad&#39;s ftp site)</td>
    </tr>    <tr>
      <td class="packagename">hg19-microsatellites-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Microsatellites from UCSC. Region which tend to be highly polymorphic...</td>
    </tr>    <tr>
      <td class="packagename">hg19-ncbi-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The composite NCBI RefSeq gene track from UCSC converted to bed format....</td>
    </tr>    <tr>
      <td class="packagename">hg19-pfam-domains-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">High quality, manually curated Pfam domain annotation in bed12 format from UCSC</td>
    </tr>    <tr>
      <td class="packagename">hg19-phastcons-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">phastCons scores for MSA of 99 genomes to hg19</td>
    </tr>    <tr>
      <td class="packagename">hg19-reference-genome-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hg19 reference genome from UCSC....</td>
    </tr>    <tr>
      <td class="packagename">hg19-repeatmasker-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">RepeatMasker track from UCSC in bed format. Interspersed repeats and...</td>
    </tr>    <tr>
      <td class="packagename">hg19-segmental-dups-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hg19 segmental duplications (SuperDups) File from UCSC. That is,...</td>
    </tr>    <tr>
      <td class="packagename">hg19-simple-repeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC. Simple tandem repeats and imperfect...</td>
    </tr>    <tr>
      <td class="packagename">hg19-simplerepeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC | name=sequence | score=alignment score...</td>
    </tr>    <tr>
      <td class="packagename">hg38-bwa-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">pre-built bwa indicies from 1000g hg38/GRCh38 distribution</td>
    </tr>    <tr>
      <td class="packagename">hg38-chrom-mapping-ensembl2ucsc-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps GRCh38...</td>
    </tr>    <tr>
      <td class="packagename">hg38-chromsizes-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for hg38</td>
    </tr>    <tr>
      <td class="packagename">hg38-chromsizes-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for the hg38 genome build from NCBI. (Used to create...</td>
    </tr>    <tr>
      <td class="packagename">hg38-cpg-islands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">hg38 cpg islands from UCSC in bed format</td>
    </tr>    <tr>
      <td class="packagename">hg38-cytobands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Approximate locations of Chromosome Bands (cytoBands) located with...</td>
    </tr>    <tr>
      <td class="packagename">hg38-gaps-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Assembly gaps from UCSC in bed fromat</td>
    </tr>    <tr>
      <td class="packagename">hg38-gtf-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh38 gtf file from ensembl remapped to hg38 (Any unmapped entries...</td>
    </tr>    <tr>
      <td class="packagename">hg38-gtf-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">hg38-microsatellites-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Microsatellites from UCSC. Region which tend to be highly polymorphic...</td>
    </tr>    <tr>
      <td class="packagename">hg38-ncbi-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The composite NCBI RefSeq gene track from UCSC converted to bed format....</td>
    </tr>    <tr>
      <td class="packagename">hg38-pfam-domains-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">High quality, manually curated Pfam domain annotation in bed12 format from UCSC</td>
    </tr>    <tr>
      <td class="packagename">hg38-reference-genome-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hg38 reference genome from UCSC. This version includes the latest...</td>
    </tr>    <tr>
      <td class="packagename">hg38-repeatmasker-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">RepeatMasker track from UCSC in bed format. Interspersed repeats and...</td>
    </tr>    <tr>
      <td class="packagename">hg38-segmental-dups-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Segmental duplications (SuperDups) File from UCSC. That is,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-sequence-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Full hg38/GRCh38 reference genome distributed by 1000 genomes Derived...</td>
    </tr>    <tr>
      <td class="packagename">hg38-simple-repeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC. Simple tandem repeats and imperfect...</td>
    </tr>    <tr>
      <td class="packagename">hg38-simplerepeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC | name=sequence | score=alignment score...</td>
    </tr>    <tr>
      <td class="packagename">mm10-reference-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Reference genome for mouse from UCSC</td>
    </tr>  </table>
  <address>Updated: 2020-04-28 19:16:36 +0000 - Files: 100</address>
</body>
</html>