<html>
<head>
  <title>ggd-genomics</title>
  <style type="text/css">
    a, a:active {
      text-decoration: none; color: blue;
    }
    a:visited {
      color: #48468F;
    }
    a:hover, a:focus {
      text-decoration: underline; color: red;
    }
    body {
      background-color: #F5F5F5;
    }
    h2 {
      margin-bottom: 12px;
    }
    th, td {
      font: 100% monospace; text-align: left;
    }
    th {
      font-weight: bold; padding-right: 14px; padding-bottom: 3px;
    }
    th.tight {
        padding-right: 6px;
    }
    td {
      padding-right: 14px;
    }
    td.tight {
        padding-right: 8px;
    }
    td.s, th.s {
      text-align: right;
    }
    td.summary {
      white-space: nowrap;
      overflow: hidden;
    }
    td.packagename {
      white-space: nowrap;
      text-overflow: ellipsis;
      overflow: hidden;
      max-width: 180px;
      padding-right: 8px;
    }
    td.version {
      //white-space: nowrap;
      overflow: hidden;
      max-width: 90px;
      padding-right: 8px;
    }
    table {
      background-color: white;
      border-top: 1px solid #646464;
      border-bottom: 1px solid #646464;
      padding-top: 10px;
      padding-bottom: 14px;
    }
    address {
      color: #787878;
      padding-top: 10px;
    }
  </style>
</head>
<body>
  <h2>ggd-genomics</h2>
  <h3><a href="rss.xml">RSS Feed</a>&nbsp;&nbsp;&nbsp;<a href="channeldata.json">channeldata.json</a></h3>
<a href="noarch">noarch</a>&nbsp;&nbsp;&nbsp;  <table>
    <tr>
      <th style="padding-right:18px;">Package</th>
      <th>Latest Version</th>
      <th>Doc</th>
      <th>Dev</th>
      <th>License</th>
<th class="tight">noarch</th>      <th>Summary</th>
    </tr>
    <tr>
      <td class="packagename">danrer10-gtf-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCz10 gtf file from ensembl remapped to danRer10 (Any unmapped...</td>
    </tr>    <tr>
      <td class="packagename">danrer10-gtf-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">danrer10-reference-genome-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The danRer10 reference genome from UCSC</td>
    </tr>    <tr>
      <td class="packagename">danrer11-reference-genome-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The danRer11 reference genome from UCSC</td>
    </tr>    <tr>
      <td class="packagename">grch37-1000g-omni-snps-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The 1000G omni snps vcf file</td>
    </tr>    <tr>
      <td class="packagename">grch37-1000g-snps-phase1-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The 1000G high confidence snps from phase 1 in vcf format</td>
    </tr>    <tr>
      <td class="packagename">grch37-amino-acid-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Protein-coding transcript translation sequences. Amino acid sequences...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-dominant-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-dominant-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-dominant-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-recessive-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-recessive-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch37-autosomal-recessive-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch37-cancer-genes-futreal-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch37-canonical-isoforms-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">UCSC defined Canonical Isoform for each gene (or cluster). This tends...</td>
    </tr>    <tr>
      <td class="packagename">grch37-canonical-transcript-features-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features from Ensembl Release-75 for protein-coding canonical...</td>
    </tr>    <tr>
      <td class="packagename">grch37-canonical-transcript-features-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Protein Coding Canonical Transcript Features for each protein coding...</td>
    </tr>    <tr>
      <td class="packagename">grch37-ccdg-sv-vcf-ccdg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2 from ccdg in grch37</td>
    </tr>    <tr>
      <td class="packagename">grch37-ccds-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Consensus Coding Seqeunce (CCDS) high confidence gene annotation from...</td>
    </tr>    <tr>
      <td class="packagename">grch37-chrom-mapping-ensembl2refseq-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps GRCh37...</td>
    </tr>    <tr>
      <td class="packagename">grch37-chrom-mapping-refseq2ensembl-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps GRCh37 RefSeq...</td>
    </tr>    <tr>
      <td class="packagename">grch37-chrom-mapping-ucsc2ensembl-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps hg19 UCSC...</td>
    </tr>    <tr>
      <td class="packagename">grch37-chrom-mapping-ucsc2refseq-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps hg19 UCSC...</td>
    </tr>    <tr>
      <td class="packagename">grch37-chromsizes-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for GRCh37</td>
    </tr>    <tr>
      <td class="packagename">grch37-chromsizes-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for the GRCh37 genome build from NCBI. (Used to...</td>
    </tr>    <tr>
      <td class="packagename">grch37-clinically-associated-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">**Liftover** All known clinically associated variants at the time of...</td>
    </tr>    <tr>
      <td class="packagename">grch37-constrained-coding-regions-quinlan-lab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Constrained Coding Regions (CCRs):...</td>
    </tr>    <tr>
      <td class="packagename">grch37-cpg-islands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">cpg islands from UCSC in bed format. Remapped from UCSC hg19 to Ensembl...</td>
    </tr>    <tr>
      <td class="packagename">grch37-cytobands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Approximate locations of Chromosome Bands (cytoBands) located with...</td>
    </tr>    <tr>
      <td class="packagename">grch37-dbsnp-138-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The dbsnp version 138 vcf file</td>
    </tr>    <tr>
      <td class="packagename">grch37-eiee-genes-ostrander-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch37-esp-variants-uw-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The NHLBI Exome Sequencing Project (ESP) variants from the University...</td>
    </tr>    <tr>
      <td class="packagename">grch37-evofold-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The evolutionary conserved mRNA secondary structure predictions hosted...</td>
    </tr>    <tr>
      <td class="packagename">grch37-gaps-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Assembly gaps from UCSC in bed fromat. Remapped from UCSC hg19 to...</td>
    </tr>    <tr>
      <td class="packagename">grch37-gene-features-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features, including all annotated transcripts, that make up the...</td>
    </tr>    <tr>
      <td class="packagename">grch37-gene-features-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Comprehensive set of gene anntotations created on GRCh38 and mapped to...</td>
    </tr>    <tr>
      <td class="packagename">grch37-gene-features-introns-only-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Intron features created from the Ensembl gene features, including all...</td>
    </tr>    <tr>
      <td class="packagename">grch37-gene-features-with-introns-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features, including all annotated transcripts, that make up the...</td>
    </tr>    <tr>
      <td class="packagename">grch37-gene-features-with-introns-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Comprehensive set of gene anntotations created on GRCh38 and mapped to...</td>
    </tr>    <tr>
      <td class="packagename">grch37-germline-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known germline variants at the time of Ensembl release 75....</td>
    </tr>    <tr>
      <td class="packagename">grch37-gtf-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh37 gtf file from ensembl</td>
    </tr>    <tr>
      <td class="packagename">grch37-haploinsufficient-genes-clingen-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch37-haploinsufficient-genes-dang-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch37-hapmap-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hapmap vcf versoin 3.3 from the broad institute</td>
    </tr>    <tr>
      <td class="packagename">grch37-heart-genes-sifrim-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch37-hg002-pb-chr22-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">GiaB PacBio 70x sequencing for chromosome 22 in ashkenazi sample HG002</td>
    </tr>    <tr>
      <td class="packagename">grch37-hg002-svs-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset for the Ashkenazi trio son HG002 in grch37</td>
    </tr>    <tr>
      <td class="packagename">grch37-hg003-pb-chr22-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">GiaB PacBio 30x sequencing for chromosome 22 in ashkenazi sample HG003</td>
    </tr>    <tr>
      <td class="packagename">grch37-hg004-pb-chr22-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">GiaB PacBio 30x sequencing for chromosome 22 in ashkenazi sample HG004</td>
    </tr>    <tr>
      <td class="packagename">grch37-known-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The &#39;KnownGene&#39; track from UCSC. Comprsied of gene predictions...</td>
    </tr>    <tr>
      <td class="packagename">grch37-knowngene-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features from the &#39;KnownGene&#39; track from UCSC. (GTF)....</td>
    </tr>    <tr>
      <td class="packagename">grch37-liftover-chain-38-to-37-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Liftover chain file from ensembl</td>
    </tr>    <tr>
      <td class="packagename">grch37-lncrna-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Long non-coding RNA gene annotation. It contains the comprehensive gene...</td>
    </tr>    <tr>
      <td class="packagename">grch37-lncrna-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Long non-coding RNA transcript sequences for CHR regions. Nucleotide...</td>
    </tr>    <tr>
      <td class="packagename">grch37-microsatellites-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Microsatellites from UCSC. Region which tend to be highly polymorphic...</td>
    </tr>    <tr>
      <td class="packagename">grch37-missense-z-score-exac-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Missense Z Score by gene. A constraint score based on missense...</td>
    </tr>    <tr>
      <td class="packagename">grch37-ncbi-refseq-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features for all curated and predicted annotations from the NCBI...</td>
    </tr>    <tr>
      <td class="packagename">grch37-ncbi-refseq-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">NCBI Refseq Genes track from UCSC. Curated and predicated genes from...</td>
    </tr>    <tr>
      <td class="packagename">grch37-pfam-domains-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">High quality, manually curated Pfam domain annotation in bed12 format...</td>
    </tr>    <tr>
      <td class="packagename">grch37-phase3-vcf-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The phase3 1000g vcf file distrubted on 05/02/2013 in vcf format...</td>
    </tr>    <tr>
      <td class="packagename">grch37-phastcons-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Conservation scores based on the phastcons HMM algorithm against...</td>
    </tr>    <tr>
      <td class="packagename">grch37-phenotype-associated-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">**Liftover** All variants at the time of Ensembl release 99 that have a...</td>
    </tr>    <tr>
      <td class="packagename">grch37-pli-scores-exac-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The probability of being loss-of-function intolerant (pLI) by gene...</td>
    </tr>    <tr>
      <td class="packagename">grch37-radar-li-lab-stanford-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Rigorously Annotated Database of A-to-I RNA Editing (RADAR). A...</td>
    </tr>    <tr>
      <td class="packagename">grch37-reference-genome-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">GRCh37 reference genome from 1000 genomes</td>
    </tr>    <tr>
      <td class="packagename">grch37-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh37 unmasked genomic DNA seqeunce reference genome from...</td>
    </tr>    <tr>
      <td class="packagename">grch37-reference-genome-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh37 DNA nucleotide sequence primary assembly. Sequence regions...</td>
    </tr>    <tr>
      <td class="packagename">grch37-refseq-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">grch37-repeatmasker-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">RepeatMasker track from UCSC in bed format. Interspersed repeats and...</td>
    </tr>    <tr>
      <td class="packagename">grch37-scores-by-amino-acid-change-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">missense badnass, blosum, grantham scores foreach amino acid change</td>
    </tr>    <tr>
      <td class="packagename">grch37-segmental-dups-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Segmental duplications (SuperDups) File from UCSC. That is,...</td>
    </tr>    <tr>
      <td class="packagename">grch37-self-chain-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Self chain alignemnts of the human genome with an improved gap scoring...</td>
    </tr>    <tr>
      <td class="packagename">grch37-sequence-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The fasta sequence from 1000g</td>
    </tr>    <tr>
      <td class="packagename">grch37-simple-repeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC. Simple tandem repeats and imperfect...</td>
    </tr>    <tr>
      <td class="packagename">grch37-somatic-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known somatic mutations at the time of Ensembl release 75....</td>
    </tr>    <tr>
      <td class="packagename">grch37-structural-variants-bedpe-ccdg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2 from ccdg in grch37, bedpe format</td>
    </tr>    <tr>
      <td class="packagename">grch37-structural-variants-database-genomic-variants-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Database of Genomic Variants incorporating dbVar, July 2013 and later</td>
    </tr>    <tr>
      <td class="packagename">grch37-structural-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known structural mutations at the time of Ensembl release 75....</td>
    </tr>    <tr>
      <td class="packagename">grch37-structural-variants-gnomad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2.1 from gnomAD</td>
    </tr>    <tr>
      <td class="packagename">grch37-sv-gnomad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset from gnomAD</td>
    </tr>    <tr>
      <td class="packagename">grch37-toplevel-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh37 unmasked genomic DNA seqeunce reference genome from...</td>
    </tr>    <tr>
      <td class="packagename">grch37-transcript-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Transcript sequences. Nucleotide sequences of all transcripts on the...</td>
    </tr>    <tr>
      <td class="packagename">grch37-transcript-sequences-protein-coding-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Nucleotide sequences of protein-coding transcripts on the reference chromosomes</td>
    </tr>    <tr>
      <td class="packagename">grch37-trna-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">tRNA gene predictions from tRNAscan-SE v1.23 hosted on UCSC. Remapped...</td>
    </tr>    <tr>
      <td class="packagename">grch37-vcf-2.1-exome-gnomad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The gnomAD whole exome vcf file. Version 2.1.1 released on March 6,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-amino-acid-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Protein-coding transcript translation sequences. Amino acid sequences...</td>
    </tr>    <tr>
      <td class="packagename">grch38-autosomal-dominant-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-autosomal-dominant-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-autosomal-dominant-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-autosomal-recessive-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-autosomal-recessive-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-autosomal-recessive-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-cancer-genes-futreal-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch38-canonical-isoforms-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The Canonical Isofrom for each gene (or cluster) from UCSC. The APPRIS...</td>
    </tr>    <tr>
      <td class="packagename">grch38-canonical-transcript-features-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features from Ensembl Release-100 for protein-coding canonical...</td>
    </tr>    <tr>
      <td class="packagename">grch38-canonical-transcript-features-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Protein Coding Canonical Transcript Features for each protein coding...</td>
    </tr>    <tr>
      <td class="packagename">grch38-ccdg-sv-vcf-ccdg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2 from ccdg in grch38</td>
    </tr>    <tr>
      <td class="packagename">grch38-ccds-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Consensus Coding Seqeunce (CCDS) high confidence gene annotation from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-chrom-mapping-ensembl2refseq-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that map GRCh38...</td>
    </tr>    <tr>
      <td class="packagename">grch38-chrom-mapping-refseq2ensembl-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that map GRCh38 RefSeq...</td>
    </tr>    <tr>
      <td class="packagename">grch38-chrom-mapping-ucsc2ensembl-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that map hg38 UCSC...</td>
    </tr>    <tr>
      <td class="packagename">grch38-chrom-mapping-ucsc2refseq-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that map hg38 UCSC to...</td>
    </tr>    <tr>
      <td class="packagename">grch38-chromsizes-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for GRCh38</td>
    </tr>    <tr>
      <td class="packagename">grch38-chromsizes-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for the GRCh38 genome build from NCBI. (Used to...</td>
    </tr>    <tr>
      <td class="packagename">grch38-clinically-associated-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known clinically associated variants at the time of Ensembl release...</td>
    </tr>    <tr>
      <td class="packagename">grch38-consensus-pseudogene-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Consensus pseudogenes predicted by the Yale and UCSC pipelines. 2-way...</td>
    </tr>    <tr>
      <td class="packagename">grch38-constrained-coding-regions-quinlan-lab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">*LiftOver* Constrained Coding Regions (CCRs):...</td>
    </tr>    <tr>
      <td class="packagename">grch38-cpg-islands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">cpg islands from UCSC in bed format. Remapped from UCSC hg38 to Ensembl...</td>
    </tr>    <tr>
      <td class="packagename">grch38-cytobands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Approximate locations of Chromosome Bands (cytoBands) located with...</td>
    </tr>    <tr>
      <td class="packagename">grch38-eiee-genes-ostrander-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch38-evofold-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">*Liftover* The evolutionary conserved mRNA secondary structure...</td>
    </tr>    <tr>
      <td class="packagename">grch38-gaps-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Assembly gaps from UCSC in bed fromat Remapped from UCSC hg38 to...</td>
    </tr>    <tr>
      <td class="packagename">grch38-gene-features-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features, including all annotated transcripts, that make up the...</td>
    </tr>    <tr>
      <td class="packagename">grch38-gene-features-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Comprehensive set of gene anntotations including reference chromosomes,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-gene-features-introns-only-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Intron features created from the Ensembl gene features, including all...</td>
    </tr>    <tr>
      <td class="packagename">grch38-gene-features-with-introns-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features, including all annotated transcripts, that make up the...</td>
    </tr>    <tr>
      <td class="packagename">grch38-gene-features-with-introns-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Comprehensive set of gene anntotations including reference chromosomes,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-gtf-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh38 gtf file from ensembl</td>
    </tr>    <tr>
      <td class="packagename">grch38-haploinsufficient-genes-clingen-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch38-haploinsufficient-genes-dang-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch38-heart-genes-sifrim-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">grch38-hg002-chr22-illumina-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Illumina paired-end sequencing alignments from HG002, downloaded from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-hg003-chr22-illumina-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Illumina paired-end sequencing alignments from HG003, downloaded from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-hg004-chr22-illumina-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Illumina paired-end sequencing alignments from HG004, downloaded from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-known-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The &#39;KnownGene&#39; track from UCSC. Comprised of all gene models...</td>
    </tr>    <tr>
      <td class="packagename">grch38-knowngene-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features from the &#39;KnownGene&#39; track from UCSC. (GTF)....</td>
    </tr>    <tr>
      <td class="packagename">grch38-liftover-chain-37-to-38-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Liftover chain file from ensembl</td>
    </tr>    <tr>
      <td class="packagename">grch38-lncrna-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Long non-coding RNA gene annotation. It contains the comprehensive gene...</td>
    </tr>    <tr>
      <td class="packagename">grch38-lncrna-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Long non-coding RNA transcript sequences for CHR regions. Nucleotide...</td>
    </tr>    <tr>
      <td class="packagename">grch38-microsatellites-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Microsatellites from UCSC. Region which tend to be highly polymorphic...</td>
    </tr>    <tr>
      <td class="packagename">grch38-missense-z-score-exac-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Missense Z Score by gene. A constraint score based on missense...</td>
    </tr>    <tr>
      <td class="packagename">grch38-ncbi-refseq-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features for all curated and predicted annotations from the NCBI...</td>
    </tr>    <tr>
      <td class="packagename">grch38-ncbi-refseq-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">NCBI Refseq Genes track from UCSC. Curated and predicated genes from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-pfam-domains-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">High quality, manually curated Pfam domain annotation in bed12 format...</td>
    </tr>    <tr>
      <td class="packagename">grch38-phastcons-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Conservation scores based on the phastcons HMM algorithm against...</td>
    </tr>    <tr>
      <td class="packagename">grch38-phenotype-associated-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All variants at the time of Ensembl release 99 that have a known...</td>
    </tr>    <tr>
      <td class="packagename">grch38-pli-scores-exac-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The probability of being loss-of-function intolerant (pLI) by gene...</td>
    </tr>    <tr>
      <td class="packagename">grch38-polya-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">PolyA feature annotation. It contains the polyA features (polyA_signal,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-predicted-trna-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Predicted tRNA genes. tRNA genes predicted by ENSEMBL on the reference...</td>
    </tr>    <tr>
      <td class="packagename">grch38-radar-li-lab-stanford-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">**Liftover** Rigorously Annotated Database of A-to-I RNA Editing...</td>
    </tr>    <tr>
      <td class="packagename">grch38-reference-genome-all-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genome sequence. Nucleotide sequence of the GRCh38.p13 genome assembly...</td>
    </tr>    <tr>
      <td class="packagename">grch38-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh38 unmasked genomic DNA seqeunce reference genome from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-reference-genome-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genome sequence, primary assembly. Nucleotide sequence of the GRCh38...</td>
    </tr>    <tr>
      <td class="packagename">grch38-refseq-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">grch38-repeatmasker-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">RepeatMasker track from UCSC in bed format. Interspersed repeats and...</td>
    </tr>    <tr>
      <td class="packagename">grch38-segmental-dups-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Segmental duplications (SuperDups) File from UCSC. That is,...</td>
    </tr>    <tr>
      <td class="packagename">grch38-self-chain-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Self chain alignemnts of the human genome with an improved gap scoring...</td>
    </tr>    <tr>
      <td class="packagename">grch38-simple-repeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC. Simple tandem repeats and imperfect...</td>
    </tr>    <tr>
      <td class="packagename">grch38-somatic-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known somatic mutations at the time of Ensembl release 99....</td>
    </tr>    <tr>
      <td class="packagename">grch38-structural-variants-bedpe-ccdg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2 from ccdg in hg38, bedpe format</td>
    </tr>    <tr>
      <td class="packagename">grch38-structural-variants-database-genomic-variants-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Database of Genomic Variants incorporating dbVar, July 2013 and later</td>
    </tr>    <tr>
      <td class="packagename">grch38-structural-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known structural mutations at the time of Ensembl release 99....</td>
    </tr>    <tr>
      <td class="packagename">grch38-structural-variants-gnomad-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2.1 from gnomAD, lifted over to build 38 in NCBI</td>
    </tr>    <tr>
      <td class="packagename">grch38-toplevel-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh38 unmasked genomic DNA seqeunce reference genome from...</td>
    </tr>    <tr>
      <td class="packagename">grch38-transcript-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Transcript sequences. Nucleotide sequences of all transcripts on the...</td>
    </tr>    <tr>
      <td class="packagename">grch38-transcript-sequences-protein-coding-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Nucleotide sequences of protein-coding transcripts on the reference chromosomes</td>
    </tr>    <tr>
      <td class="packagename">grch38-trna-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">tRNA gene predictions from tRNAscan-SE v1.23 hosted on UCSC. Remapped...</td>
    </tr>    <tr>
      <td class="packagename">grcz10-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCz10 release 91 reference genome from Ensembl</td>
    </tr>    <tr>
      <td class="packagename">grcz11-reference-genome-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCz11 release 96 reference genome from Ensembl</td>
    </tr>    <tr>
      <td class="packagename">hg19-1000g-omni-snps-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The 1000G omni snps vcf file hosted on the broad&#39;s ftp site</td>
    </tr>    <tr>
      <td class="packagename">hg19-1000g-snps-phase1-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The 1000G high confidence snps from phase 1 in vcf format. (Hosted on...</td>
    </tr>    <tr>
      <td class="packagename">hg19-amino-acid-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Protein-coding transcript translation sequences. Amino acid sequences...</td>
    </tr>    <tr>
      <td class="packagename">hg19-autosomal-dominant-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg19-autosomal-dominant-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg19-autosomal-dominant-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg19-autosomal-recessive-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg19-autosomal-recessive-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg19-autosomal-recessive-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg19-cancer-genes-futreal-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg19-canonical-isoforms-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">UCSC defined Canonical Isoform for each gene (or cluster). This tends...</td>
    </tr>    <tr>
      <td class="packagename">hg19-canonical-transcript-features-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features from Ensembl Release-75 for protein-coding canonical...</td>
    </tr>    <tr>
      <td class="packagename">hg19-canonical-transcript-features-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Protein Coding Canonical Transcript Features for each protein coding...</td>
    </tr>    <tr>
      <td class="packagename">hg19-ccdg-sv-vcf-ccdg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2 from ccdg in hg19</td>
    </tr>    <tr>
      <td class="packagename">hg19-ccds-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Consensus Coding Seqeunce (CCDS) high confidence gene annotation from...</td>
    </tr>    <tr>
      <td class="packagename">hg19-chrom-mapping-ensembl2ucsc-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps GRCh37...</td>
    </tr>    <tr>
      <td class="packagename">hg19-chrom-mapping-refseq2ucsc-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps GRCh37 RefSeq...</td>
    </tr>    <tr>
      <td class="packagename">hg19-chromsizes-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for hg19</td>
    </tr>    <tr>
      <td class="packagename">hg19-chromsizes-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for the hg19 genome build from NCBI. (Used to create...</td>
    </tr>    <tr>
      <td class="packagename">hg19-clinically-associated-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">**Liftover** All known clinically associated variants at the time of...</td>
    </tr>    <tr>
      <td class="packagename">hg19-constrained-coding-regions-quinlan-lab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Constrained Coding Regions (CCRs):...</td>
    </tr>    <tr>
      <td class="packagename">hg19-cpg-islands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">cpg islands from UCSC in bed format. Scaffoldings that are not...</td>
    </tr>    <tr>
      <td class="packagename">hg19-cytobands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Approximate locations of Chromosome Bands (cytoBands) located with...</td>
    </tr>    <tr>
      <td class="packagename">hg19-dbsnp-138-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The dbsnp version 138 vcf file (hosted on the broad&#39;s ftp site)</td>
    </tr>    <tr>
      <td class="packagename">hg19-eiee-genes-ostrander-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg19-evofold-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The evolutionary conserved mRNA secondary structure predictions hosted...</td>
    </tr>    <tr>
      <td class="packagename">hg19-gaps-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Assembly gaps from UCSC in bed fromat. Scaffoldings that are not...</td>
    </tr>    <tr>
      <td class="packagename">hg19-gene-features-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features, including all annotated transcripts, that make up the...</td>
    </tr>    <tr>
      <td class="packagename">hg19-gene-features-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Comprehensive set of gene anntotations created on GRCh38 and mapped to...</td>
    </tr>    <tr>
      <td class="packagename">hg19-gene-features-introns-only-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Intron features created from the Ensembl gene features, including all...</td>
    </tr>    <tr>
      <td class="packagename">hg19-gene-features-with-introns-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features, including all annotated transcripts, that make up the...</td>
    </tr>    <tr>
      <td class="packagename">hg19-gene-features-with-introns-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Comprehensive set of gene anntotations created on GRCh38 and mapped to...</td>
    </tr>    <tr>
      <td class="packagename">hg19-germline-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known germline variants at the time of Ensembl release 75....</td>
    </tr>    <tr>
      <td class="packagename">hg19-gtf-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">hg19-haploinsufficient-genes-clingen-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg19-haploinsufficient-genes-dang-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg19-hapmap-broad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hapmap vcf versoin 3.3 (hosted on the  broad&#39;s ftp site)</td>
    </tr>    <tr>
      <td class="packagename">hg19-heart-genes-sifrim-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg19-hg002-svs-giab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset for the Ashkenazi trio son HG002 in hg19</td>
    </tr>    <tr>
      <td class="packagename">hg19-known-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The &#39;KnownGene&#39; track from UCSC. Comprsied of gene predictions...</td>
    </tr>    <tr>
      <td class="packagename">hg19-knowngene-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features from the &#39;KnownGene&#39; track from UCSC. (GTF)....</td>
    </tr>    <tr>
      <td class="packagename">hg19-liftover-chain-38-to-19-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Liftover chain file from UCSC</td>
    </tr>    <tr>
      <td class="packagename">hg19-lncrna-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Long non-coding RNA gene annotation. It contains the comprehensive gene...</td>
    </tr>    <tr>
      <td class="packagename">hg19-lncrna-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Long non-coding RNA transcript sequences for CHR regions. Nucleotide...</td>
    </tr>    <tr>
      <td class="packagename">hg19-microsatellites-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Microsatellites from UCSC. Region which tend to be highly polymorphic...</td>
    </tr>    <tr>
      <td class="packagename">hg19-missense-z-score-exac-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Missense Z Score by gene. A constraint score based on missense...</td>
    </tr>    <tr>
      <td class="packagename">hg19-ncbi-refseq-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features for all curated and predicted annotations from the NCBI...</td>
    </tr>    <tr>
      <td class="packagename">hg19-ncbi-refseq-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">NCBI Refseq Genes track from UCSC. Curated and predicated genes from...</td>
    </tr>    <tr>
      <td class="packagename">hg19-ncbi-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The composite NCBI RefSeq gene track from UCSC converted to bed format....</td>
    </tr>    <tr>
      <td class="packagename">hg19-pfam-domains-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">High quality, manually curated Pfam domain annotation in bed12 format from UCSC</td>
    </tr>    <tr>
      <td class="packagename">hg19-phastcons-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Conservation scores based on the phastcons HMM algorithm against...</td>
    </tr>    <tr>
      <td class="packagename">hg19-phenotype-associated-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">**Liftover** All variants at the time of Ensembl release 99 that have a...</td>
    </tr>    <tr>
      <td class="packagename">hg19-pli-scores-exac-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The probability of being loss-of-function intolerant (pLI) by gene...</td>
    </tr>    <tr>
      <td class="packagename">hg19-radar-li-lab-stanford-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Rigorously Annotated Database of A-to-I RNA Editing (RADAR). A...</td>
    </tr>    <tr>
      <td class="packagename">hg19-reference-genome-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh37 DNA nucleotide sequence primary assembly. Sequence regions...</td>
    </tr>    <tr>
      <td class="packagename">hg19-reference-genome-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hg19 soft masked genomic DNA seqeunce reference genome from UCSC...</td>
    </tr>    <tr>
      <td class="packagename">hg19-refseq-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">hg19-repeatmasker-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">RepeatMasker track from UCSC in bed format. Interspersed repeats and...</td>
    </tr>    <tr>
      <td class="packagename">hg19-segmental-dups-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hg19 segmental duplications (SuperDups) File from UCSC. That is,...</td>
    </tr>    <tr>
      <td class="packagename">hg19-self-chain-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Self chain alignemnts of the human genome with an improved gap scoring...</td>
    </tr>    <tr>
      <td class="packagename">hg19-simple-repeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC. Simple tandem repeats and imperfect...</td>
    </tr>    <tr>
      <td class="packagename">hg19-somatic-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known somatic mutations at the time of Ensembl release 75....</td>
    </tr>    <tr>
      <td class="packagename">hg19-structural-variants-bedpe-ccdg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2 from ccdg in grch37, bedpe format</td>
    </tr>    <tr>
      <td class="packagename">hg19-structural-variants-database-genomic-variants-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Database of Genomic Variants incorporating dbVar, July 2013 and later</td>
    </tr>    <tr>
      <td class="packagename">hg19-structural-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known structural mutations at the time of Ensembl release 75....</td>
    </tr>    <tr>
      <td class="packagename">hg19-structural-variants-gnomad-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2.1 from gnomAD. Remapped from Ensembl GRCh37 to UCSC hg19</td>
    </tr>    <tr>
      <td class="packagename">hg19-transcript-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Transcript sequences. Nucleotide sequences of all transcripts on the...</td>
    </tr>    <tr>
      <td class="packagename">hg19-transcript-sequences-protein-coding-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Nucleotide sequences of protein-coding transcripts on the reference chromosomes</td>
    </tr>    <tr>
      <td class="packagename">hg19-trna-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">tRNA gene predictions from tRNAscan-SE v1.23 hosted on UCSC.</td>
    </tr>    <tr>
      <td class="packagename">hg38-amino-acid-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Protein-coding transcript translation sequences. Amino acid sequences...</td>
    </tr>    <tr>
      <td class="packagename">hg38-autosomal-dominant-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-autosomal-dominant-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-autosomal-dominant-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-autosomal-recessive-genes-berg-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-autosomal-recessive-genes-berg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-autosomal-recessive-genes-blekhman-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with the compliment coordinates,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-bwa-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">pre-built bwa indicies from 1000g hg38/GRCh38 distribution</td>
    </tr>    <tr>
      <td class="packagename">hg38-cancer-genes-futreal-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg38-canonical-isoforms-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The Canonical Isofrom for each gene (or cluster) from UCSC. The APPRIS...</td>
    </tr>    <tr>
      <td class="packagename">hg38-canonical-transcript-features-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features from Ensembl Release-100 for protein-coding canonical...</td>
    </tr>    <tr>
      <td class="packagename">hg38-canonical-transcript-features-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Protein Coding Canonical Transcript Features for each protein coding...</td>
    </tr>    <tr>
      <td class="packagename">hg38-ccdg-sv-vcf-ccdg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2 from ccdg in hg38</td>
    </tr>    <tr>
      <td class="packagename">hg38-ccds-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Consensus Coding Seqeunce (CCDS) high confidence gene annotation from...</td>
    </tr>    <tr>
      <td class="packagename">hg38-chrom-mapping-ensembl2ucsc-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that maps GRCh38...</td>
    </tr>    <tr>
      <td class="packagename">hg38-chrom-mapping-refseq2ucsc-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">A tab delimited file containing scaffolding ids that map GRCh38 RefSeq...</td>
    </tr>    <tr>
      <td class="packagename">hg38-chromsizes-ggd-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for hg38</td>
    </tr>    <tr>
      <td class="packagename">hg38-chromsizes-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Chromosome lengths for the hg38 genome build from NCBI. (Used to create...</td>
    </tr>    <tr>
      <td class="packagename">hg38-clinically-associated-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known clinically associated variants at the time of Ensembl release...</td>
    </tr>    <tr>
      <td class="packagename">hg38-consensus-pseudogene-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Consensus pseudogenes predicted by the Yale and UCSC pipelines. 2-way...</td>
    </tr>    <tr>
      <td class="packagename">hg38-constrained-coding-regions-quinlan-lab-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">*LiftOver* Constrained Coding Regions (CCRs):...</td>
    </tr>    <tr>
      <td class="packagename">hg38-cpg-islands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">hg38 cpg islands from UCSC in bed format</td>
    </tr>    <tr>
      <td class="packagename">hg38-cytobands-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Approximate locations of Chromosome Bands (cytoBands) located with...</td>
    </tr>    <tr>
      <td class="packagename">hg38-eiee-genes-ostrander-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg38-evofold-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">*Liftover* The evolutionary conserved mRNA secondary structure...</td>
    </tr>    <tr>
      <td class="packagename">hg38-gaps-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Assembly gaps from UCSC in bed fromat</td>
    </tr>    <tr>
      <td class="packagename">hg38-gene-features-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features, including all annotated transcripts, that make up the...</td>
    </tr>    <tr>
      <td class="packagename">hg38-gene-features-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Comprehensive set of gene anntotations including reference chromosomes,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-gene-features-introns-only-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Intron features created from the Ensembl gene features, including all...</td>
    </tr>    <tr>
      <td class="packagename">hg38-gene-features-with-introns-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features, including all annotated transcripts, that make up the...</td>
    </tr>    <tr>
      <td class="packagename">hg38-gene-features-with-introns-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Comprehensive set of gene anntotations including reference chromosomes,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-gtf-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The GRCh38 gtf file from ensembl remapped to hg38 (Any unmapped entries...</td>
    </tr>    <tr>
      <td class="packagename">hg38-gtf-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">hg38-haploinsufficient-genes-clingen-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg38-haploinsufficient-genes-dang-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg38-heart-genes-sifrim-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">CDS region genomic coordinates, along with complement coordinates, for...</td>
    </tr>    <tr>
      <td class="packagename">hg38-known-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The &#39;KnownGene&#39; track from UCSC. Comprised of all gene models...</td>
    </tr>    <tr>
      <td class="packagename">hg38-knowngene-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features from the &#39;KnownGene&#39; track from UCSC. (GTF)....</td>
    </tr>    <tr>
      <td class="packagename">hg38-liftover-chain-19-to-38-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Liftover chain file from UCSC</td>
    </tr>    <tr>
      <td class="packagename">hg38-lncrna-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Long non-coding RNA gene annotation. It contains the comprehensive gene...</td>
    </tr>    <tr>
      <td class="packagename">hg38-lncrna-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Long non-coding RNA transcript sequences for CHR regions. Nucleotide...</td>
    </tr>    <tr>
      <td class="packagename">hg38-microsatellites-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Microsatellites from UCSC. Region which tend to be highly polymorphic...</td>
    </tr>    <tr>
      <td class="packagename">hg38-missense-z-score-exac-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Missense Z Score by gene. A constraint score based on missense...</td>
    </tr>    <tr>
      <td class="packagename">hg38-ncbi-refseq-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Gene features for all curated and predicted annotations from the NCBI...</td>
    </tr>    <tr>
      <td class="packagename">hg38-ncbi-refseq-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">NCBI Refseq Genes track from UCSC. Curated and predicated genes from...</td>
    </tr>    <tr>
      <td class="packagename">hg38-ncbi-refseq-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The composite NCBI RefSeq gene track from UCSC converted to bed format....</td>
    </tr>    <tr>
      <td class="packagename">hg38-pfam-domains-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">High quality, manually curated Pfam domain annotation in bed12 format from UCSC</td>
    </tr>    <tr>
      <td class="packagename">hg38-phastcons-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Conservation scores based on the phastcons HMM algorithm against...</td>
    </tr>    <tr>
      <td class="packagename">hg38-phenotype-associated-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All variants at the time of Ensembl release 99 that have a known...</td>
    </tr>    <tr>
      <td class="packagename">hg38-pli-scores-exac-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The probability of being loss-of-function intolerant (pLI) by gene...</td>
    </tr>    <tr>
      <td class="packagename">hg38-polya-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">PolyA feature annotation. It contains the polyA features (polyA_signal,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-predicted-trna-annotations-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Predicted tRNA genes. tRNA genes predicted by ENSEMBL on the reference...</td>
    </tr>    <tr>
      <td class="packagename">hg38-radar-li-lab-stanford-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">**Liftover** Rigorously Annotated Database of A-to-I RNA Editing...</td>
    </tr>    <tr>
      <td class="packagename">hg38-reference-genome-all-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genome sequence. Nucleotide sequence of the hg38.p13 genome assembly...</td>
    </tr>    <tr>
      <td class="packagename">hg38-reference-genome-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Genome sequence, primary assembly. Nucleotide sequence of the hg38...</td>
    </tr>    <tr>
      <td class="packagename">hg38-reference-genome-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The hg38 soft masked genomic DNA seqeunce reference genome from UCSC...</td>
    </tr>    <tr>
      <td class="packagename">hg38-refseq-gene-features-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">The UCSC RefSeq gene track converted into GTF format containing known...</td>
    </tr>    <tr>
      <td class="packagename">hg38-repeatmasker-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">RepeatMasker track from UCSC in bed format. Interspersed repeats and...</td>
    </tr>    <tr>
      <td class="packagename">hg38-segmental-dups-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Segmental duplications (SuperDups) File from UCSC. That is,...</td>
    </tr>    <tr>
      <td class="packagename">hg38-self-chain-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Self chain alignemnts of the human genome with an improved gap scoring...</td>
    </tr>    <tr>
      <td class="packagename">hg38-sequence-1000g-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Full hg38/GRCh38 reference genome distributed by 1000 genomes Derived...</td>
    </tr>    <tr>
      <td class="packagename">hg38-simple-repeats-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Simple repeats track from UCSC. Simple tandem repeats and imperfect...</td>
    </tr>    <tr>
      <td class="packagename">hg38-somatic-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known somatic mutations at the time of Ensembl release 99....</td>
    </tr>    <tr>
      <td class="packagename">hg38-structural-variants-bedpe-ccdg-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2 from ccdg in hg38, bedpe format</td>
    </tr>    <tr>
      <td class="packagename">hg38-structural-variants-database-genomic-variants-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Database of Genomic Variants incorporating dbVar, July 2013 and later</td>
    </tr>    <tr>
      <td class="packagename">hg38-structural-variants-ensembl-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">All known structural mutations at the time of Ensembl release 99....</td>
    </tr>    <tr>
      <td class="packagename">hg38-structural-variants-gnomad-ncbi-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">SV callset 2.1 from gnomAD, lifted over to build 38 in NCBI. Remapped...</td>
    </tr>    <tr>
      <td class="packagename">hg38-transcript-sequences-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Transcript sequences. Nucleotide sequences of all transcripts on the...</td>
    </tr>    <tr>
      <td class="packagename">hg38-transcript-sequences-protein-coding-chr-regions-gencode-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Nucleotide sequences of protein-coding transcripts on the reference chromosomes</td>
    </tr>    <tr>
      <td class="packagename">hg38-trna-genes-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">tRNA gene predictions from tRNAscan-SE v1.23 hosted on UCSC.</td>
    </tr>    <tr>
      <td class="packagename">mm10-reference-ucsc-v1</td>
      <td class="version">1</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td>      <td class="summary">Reference genome for mouse from UCSC</td>
    </tr>  </table>
  <address>Updated: 2020-08-22 18:46:30 +0000 - Files: 299</address>
</body>
</html>